Initial attempt at adding usage docs (#50)
nf-core/taxprofiler: README.md

1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
2. Performs optional read pre-processing
   - Adapter clipping and merging (short read: [fastp](https://github.com/OpenGene/fastp), [AdapterRemoval2](https://github.com/MikkelSchubert/adapterremoval); long read: [porechop](https://github.com/rrwick/Porechop))
   - Low complexity filtering ([bbduk](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/), [PRINSEQ++](https://github.com/Adrian-Cantu/PRINSEQ-plus-plus))
   - Host read removal ([BowTie2](http://bowtie-bio.sourceforge.net/bowtie2/))
   - Run merging

> 3. Performs taxonomic profiling via a choice of any or all of:

Reviewer: ```suggestion
3. Performs taxonomic profiling using one or more of:
```
